backend/db/database_test.go

```diff
@@ -82,7 +82,18 @@ func TestDatabase_Tags(t *testing.T) {
 	assert.Contains(t, res1, game1)
 	assert.Contains(t, res1, game1)
 }
+// func TestRemoveGame(t *testing.T){
+// 	t.Cleanup(cleanup)
+// 	id0, _ := _db.AddGame(game0)
+// 	id1, _ := _db.AddGame(game1)
 
+// 	game0.Id = id0
+// 	game1.Id = id1
+
+// 	res0, _ := _db.RemoveGame(id0)
+// 	assert.Equal(t, res, game0A)
+
+// }
```

Comment:
Did you mean to commit this? We probably shouldn't commit blocks of unused, commented code.

Reply:
Good Catch. I just looked into it, this commit was an older one, the newer commits do not have this commented code. Thanks.